schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-distribution.xsl

```diff
@@ -0,0 +1,443 @@
+                          or matches($protocol, 'OGC:WMS|OGC:WFS|OGC:WCS|OGC:WPS|OGC API Features|OGC API Coverages|ESRI:REST')">
+              <dcat:accessService>
+                <rdf:Description>
+                  <rdf:type rdf:resource="http://www.w3.org/ns/dcat#DataService"/>
+                  <xsl:apply-templates mode="iso19115-3-to-dcat"
+                                       select="*/cit:name[normalize-space(.) != '']"/>
+                  <dcat:endpointURL rdf:resource="{$url}"/>
+                  <!-- TODO: GetCapabilities document
+                  <dcat:endpointDescription rdf:resource="{$endpoint-description}"/>
+                   -->
+
+                  <xsl:variable name="standardPage"
+                                as="node()?"
+                                select="$protocolToStandardPage[value = lower-case($protocol)]"/>
+                  <xsl:if test="$standardPage">
+                    <dct:conformsTo>
+                      <dct:Standard rdf:about="{$standardPage/@key}"/>
+                    </dct:conformsTo>
+                  </xsl:if>
+                </rdf:Description>
+              </dcat:accessService>
+            </xsl:if>
+
+            <!--
+            RDF Property:	dcat:mediaType
+            Definition:	The media type of the distribution as defined by IANA [IANA-MEDIA-TYPES].
+            Sub-property of:	dcterms:format
+            Domain:	dcat:Distribution
+            Range:	dcterms:MediaType
+            Usage note:	This property SHOULD be used when the media type of the distribution is defined in IANA [IANA-MEDIA-TYPES], otherwise dcterms:format MAY be used with different values.
+
+            RDF Property:	dcat:compressFormat
+            Definition:	The compression format of the distribution in which the data is contained in a compressed form, e.g., to reduce the size of the downloadable file.
+            Range:	dcterms:MediaType
+            Usage note:	This property to be used when the files in the distribution are compressed, e.g., in a ZIP file. The format SHOULD be expressed using a media type as defined by IANA [IANA-MEDIA-TYPES], if available.
+
+            RDF Property:	dcat:packageFormat
+            Definition:	The package format of the distribution in which one or more data files are grouped together, e.g., to enable a set of related files to be downloaded together.
+            Range:	dcterms:MediaType
+            Usage note:	This property to be used when the files in the distribution are packaged, e.g., in a TAR file, a ZIP file, a Frictionless Data Package or a Bagit file. The format SHOULD be expressed using a media type as defined by IANA [IANA-MEDIA-TYPES], if available.
+            See also:	6.8.18 Property: compression format.
+
+            Rule:
+            * Use mimetype if any
+            * Use WWW:DOWNLOAD:(.*=format) if any
+            * fallback to ancestor::mrd:MD_DigitalTransferOptions/mrd:distributionFormat/*/mrd:formatSpecificationCitation
```

Comment:
This is a bit unclear, apparently for `dcat:mediaType`, `dcat:compressFormat` and `dcat:packageFormat` relies on`mrd:MD_DigitalTransferOptions/mrd:distributionFormat/*/mrd:formatSpecificationCitation`?

But in https://github.com/geonetwork/core-geonetwork/pull/7600/files#diff-5fa6856eb0e0615450b025fa6ac293be136ee98f4b82b908997ebf6667438ef8R399-R404 it is not set the `elementName` for any of them.

For `dcat:compressFormat` there is another template, that is also unclear if used: https://github.com/geonetwork/core-geonetwork/pull/7600/files#diff-5fa6856eb0e0615450b025fa6ac293be136ee98f4b82b908997ebf6667438ef8R407-R413

Reply:
Not sure if pointing to https://github.com/SPW-DIG/metawal-core-geonetwork/blob/dcat/schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-distribution.xsl#L416-L417 but if yes, the template define a default value for `elementName`.